oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/mysql-service.json

```diff
@@ -0,0 +1,351 @@
+               "metricTypes":[
+                  "readMetricsValues",
+                  "readMetricsValues",
+                  "readMetricsValues",
+                  "readMetricsValues"
+               ],
+               "metrics":[
+                  "meter_mysql_threads_cached",
+                  "meter_mysql_threads_running",
+                  "meter_mysql_threads_created",
+                  "meter_mysql_threads_connected"
+               ],
+               "graph":{
+                  "type":"Area",
+                  "opacity":0.4,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Threads"
+               },
+               "value":"7",
+               "label":"7"
+            },
+            {
+               "x":0,
+               "y":28,
+               "w":8,
+               "h":15,
+               "i":"8",
+               "type":"Widget",
+               "metricTypes":[
+                  "readMetricsValues",
+                  "readMetricsValues"
+               ],
+               "metrics":[
+                  "meter_mysql_connects_available",
+                  "meter_mysql_connects_aborted"
+               ],
+               "graph":{
+                  "type":"Line",
+                  "step":false,
+                  "smooth":false,
+                  "showSymbol":false,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Connects"
+               },
+               "value":"8",
+               "label":"8"
+            },
+            {
+               "x":8,
+               "y":43,
+               "w":8,
+               "h":15,
+               "i":"9",
+               "type":"Widget",
+               "metricTypes":[
+                  "readMetricsValue"
+               ],
+               "metrics":[
+                  "meter_mysql_slow_queries"
+               ],
+               "graph":{
+                  "type":"Card",
+                  "fontSize":40,
+                  "textAlign":"center",
+                  "showUnit":true
+               },
+               "widget":{
+                  "title":"Slow Queries"
+               }
+            },
+            {
+               "x":0,
+               "y":43,
+               "w":8,
+               "h":15,
+               "i":"10",
+               "type":"Widget",
+               "metricTypes":[
+                  "readMetricsValues"
+               ],
+               "metrics":[
+                  "meter_mysql_slow_queries_rate"
+               ],
+               "graph":{
+                  "type":"Line",
+                  "step":false,
+                  "smooth":false,
+                  "showSymbol":false,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Slow Quries Rate"
```

Comment:
Rate should be replaced by `Trend` in the dashboard. As rate declares how to calculate, but trend shows the meaning of this graph.